jest.config.cjs

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  coverageDirectory: '<rootDir>/test-results/coverage',
+  collectCoverageFrom: ['src/**/*.{ts,tsx}', '!src/**/*-stories.tsx', '!src/**/index.ts'],
+  coverageReporters: ['lcovonly', 'text'],
+  moduleFileExtensions: ['ts', 'tsx', 'js'],
+  moduleNameMapper: {
+    '^.+\\.(md|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+      '<rootDir>/config/jest/FileStub.js',
+    '^.+\\.css$': '<rootDir>/config/jest/CSSStub.js',
+  },
+  setupFiles: ['<rootDir>/config/jest/SetupTestEnvironment.ts'],
+  setupFilesAfterEnv: [
+    '<rootDir>/config/jest/SetupReactTestingLibrary.ts',
+    '<rootDir>/config/jest/SetupJestAxe.ts',
+  ],
+  snapshotSerializers: ['@emotion/jest/serializer'],
+  testEnvironment: 'jsdom',
+  testPathIgnorePatterns: [
+    '<rootDir>/config',
+    '<rootDir>/design-system',
+    '<rootDir>/node_modules',
+    '<rootDir>/scripts',
+  ],
+  testRegex: '(/__tests__/.*|\\-test)\\.(ts|tsx)$',
+  transform: {
+    '^.+\\.(t|j)sx?$': [
+      '@swc/jest',
+      {
+        jsc: {
+          target: 'es2018',
+        },
+      },
+    ],
+  },
+  transformIgnorePatterns: ['/node_modules/(?!(d3-.+))/'],
+  reporters: [
+    'default',
+    [
+      'jest-junit',
```

Comment:
What's the advantage of the junit reporter ? Makes me tickle to see java related stuff on this repo

Reply:
isn't it required for SQ consumption?

Reply:
Ah I'm not sure then, SC doesn't need it that's for sure

Reply:
I didn't find anything in SQ doc talking about jest-junit, I'll drop it for now and if it's needed when we put the CI in place we can add it back